release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -47,6 +47,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-12-aws-load-balancer-customization"]
+==== Specify the load balancer type in AWS during installation
+Beginning with {product-title} {product-version}, you can specify either NLB or Classic as a persistent load balancer type in AWS during installation. Afterwards, if an ingress controller is deleted, the load balancer type persists with the lbType configured during installation. 
```

Comment:
Do we know what NLB is? I probably should known this, but I don't.

We're capitalizing Ingress Controller, although I don't recall why: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress-controller

This seems related to: https://kubernetes.io/docs/concepts/services-networking/service/#aws-nlb-support

Is `lbType` an object field?